src/main/java/com/impactupgrade/integration/hubspot/model/ContactProperties.java

```diff
@@ -6,6 +6,15 @@
   private HasValue<String> lastname;
   private HasValue<String> email;
   private HasValue<String> phone;
+  private HasValue<String> home_phone;
```

Comment:
Would rather see camelCase here + `@JsonProperty` annotations

Reply:
@JsonProperty("firstname") private HasValue<String> firstName;

This what you're looking for here? Or something like this?